docs/UserReward.rst

```diff
@@ -0,0 +1,132 @@
+.. _UserReward:
+
+#####################
+Managing your reward
```

Comment:
```suggestion
Managing your Wallet
```

Reply:
The idea is this chapter will contain the documentation of all the interfaces that are related to reward module for users : 
- User Wallet
- Kudos (for users) and
- Perkstore

Reply:
I don't think it is a good structure. They could all 3 be at the same level.
rewars is a marketing packaging name (which also oncludes gamification by the way), but wallet, kudos and perkstore are different apps.